### PR TITLE
chore: remove polyfills build

### PR DESCRIPTION
**Pull request by patrickhousley**
Please add a one-paragraph summary here, suitable for a release notes description. This will help with documentation.
---

### Overview

Removing the IE11 polyfills build.

### Related Issue(s)

https://new-relic.atlassian.net/browse/NR-298895

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -205,7 +205,6 @@
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/plugin-proposal-private-methods": "^7.18.6",
-    "@babel/plugin-syntax-import-assertions": "^7.23.3",
```
Comment (metal-messiah) [inline package.json:208]: can you also remove `core-js` in this PR

Comment (patrickhousley) [reply package.json:208]: yup forgot about that

**Review by metal-messiah — APPROVED** [review]
Approved. just a reminder that the server-side polyfill pinning needs to be fully implemented in all production envs before releasing this to customers.

**Comment by patrickhousley** [comment]
> Approved. just a reminder that the server-side polyfill pinning needs to be fully implemented in all production envs before releasing this to customers.

The server side has been deployed to all prod and eu-prod pods.

**Review by metal-messiah — APPROVED** [review]

**Pull request merged**